test/blackbox-tests/test-cases/pkg/git-repo.t

```diff
@@ -74,15 +74,15 @@ should also be included.
 Locking should be successful and it should include the additional file
 
   $ XDG_CACHE_HOME=$PWD/dune-cache dune pkg lock
-  Solution for dune.lock:
+  Solution for .dune-solution-cache:
   - foo.1.2
 
   $ find ${default_lock_dir} | sort
```

Comment:
Maybe adding something like `| sed "s/^${default_lock_dir}/LOCK_DIR/"` could be nice to avoid changing output could be a good idea here.